cmd/minikube/cmd/start.go

```diff
@@ -850,3 +853,52 @@ func saveConfig(clusterConfig cfg.Config) error {
 	}
 	return nil
 }
+
+func validateDriverVersion(vmDriver string) {
+	if vmDriver == constants.DriverKvm2 {
+		cmd := exec.Command("docker-machine-driver-kvm2", "version")
+		output, err := cmd.Output()
+
+		// we don't want to fail if an error was returned,
+		// libmachine has a nice message for the user if the driver isn't present
+		if err != nil {
+			console.Warning("Error checking driver version: %v", err)
+			return
+		}
+
+		v := extractVMDriverVersion(string(output))
+
+		// if the driver doesn't have return any version, it is really old, we force a upgrade.
+		if len(v) == 0 {
+			exit.WithCode(exit.Failure, "Please upgrade the 'docker-machine-driver-kvm2'.")
+		}
+
+		vmDriverVersion, err := semver.Make(v)
+		if err != nil {
+			console.Warning("Error parsing vmDriver version: %v", err)
+			return
+		}
+
+		minikubeVersion, err := version.GetSemverVersion()
+		if err != nil {
+			console.Warning("Error parsing minukube version: %v", err)
+			return
+		}
+
+		if vmDriverVersion.LT(minikubeVersion) {
+			console.Warning("The 'docker-machine-driver-kvm2' version is old. Please consider upgrading.")
```

Comment:
same here, we could link to a page where they can easily find the newest kvm2.

( i noticed our docs page is a bit confusion, as part of this PR we could make that doc page for kvm2 super simple.( we have too many ifs in the doc)